nexus/src/app/support_bundles.rs

```diff
@@ -90,7 +90,21 @@ impl super::Nexus {
             .db_datastore
             .zpool_get_sled_if_in_service(&opctx, bundle.zpool_id.into())
             .await?;
-        let client = self.sled_client(&sled_id).await?;
+
+        let short_timeout = std::time::Duration::from_secs(60);
+        let long_timeout = std::time::Duration::from_secs(3600);
+        let client = reqwest::ClientBuilder::new()
+            // Connecting and continuing to read from the sled agent should
+            // happen relatively quickly.
+            .connect_timeout(short_timeout)
+            .read_timeout(short_timeout)
+            // However, the bundle itself may be large. As long as we're
+            // continuing to make progress (see: read_timeout) we should be
+            // willing to keep transferring the bundle for a while longer.
+            .timeout(long_timeout)
```

Comment:
This may still be too short for a truly large support bundle, say 50 GiB. At a download speed of 10 MiB/s, that would take roughly 85 minutes to complete.

Reply:
I hear you - I think range requests is how I want to solve this in the limit; I'm not sure I really want to allow connections to stay open in Nexus for longer than an hour.

WDYT - should we still try to do this PR as-is? Do you think we should aim for a higher number? Or should we accept this limitation until range requests are fully plumbed through?